client/src/lib/i18n/index.ts

```diff
@@ -18,20 +18,39 @@ import {
   DEFAULT_LANGUAGE_TAG,
   LANGUAGE_TAGS,
 } from '../../../../shared/src/constants/i18n';
+import Backend from './backend/backend';
+import {clone} from 'ramda';
 
 export * from '../../../../shared/src/constants/i18n';
 
 dayjs.extend(localizedFormat);
 
 const DEFAULT_24HOUR_LANGUAGE_TAG = 'en-gb';
 
+// To trigger the backend middleware to load exercises they have to be removed first.
+// Removing them in buildContent creates somewhat of a mess in backend adding them back.
+const omitExercises = (resources: typeof content.i18n) => {
+  const allResources = clone(resources) as unknown as Record<
+    string,
+    Record<string, string>
+  >; // Can't delete exercises in type content.i18n since it's not opttional
+  for (const ln of LANGUAGE_TAGS) {
+    delete allResources[ln].exercises;
+  }
+  return allResources;
+};
```

Comment:
Nit pick, how about moving this function and `useAppState.getState().showHiddenContent` to a utils file?

Reply:
Will do